mitosheet/mitosheet/step_performers/step_performer.py

```diff
@@ -61,18 +67,21 @@ def execute(cls, prev_state: State, params: Dict[str, Any]) -> Tuple[State, Opti
         will allow the logging infrastructure to determine how much overhead Mito
         adds to executing this event.
 
-        If the exectution_data includes the key `result`, than this result can be
-        accessed in the frontend to display some results to the user. See the 
-        DropDuplicatesStepPerformer and the DropDuplicatesCodeChunk for an example of this usage.
+        By default, this function will return the result of just executing the transpiled
+        code. 
+
+        In general, this function should use _no_ Pandas specific functionality, but rather 
```

Comment:
Currently, we still have some pandas specific code for error handing like in merge btw

Reply:
Yeah. We can keep refining this over time, but it's all good for now.